vector/src/main/java/im/vector/app/features/voicebroadcast/listening/VoiceBroadcastPlayerImpl.kt

```diff
@@ -302,15 +315,53 @@ class VoiceBroadcastPlayerImpl @Inject constructor(
     }
 
     private fun onPlayingStateChanged(playingState: State) {
-        // Notify state change to all the listeners attached to the current voice broadcast id
+        // Update live playback flag
+        updateLiveListeningMode()
+
         currentVoiceBroadcast?.voiceBroadcastId?.let { voiceBroadcastId ->
+            // Start or stop playback ticker
             when (playingState) {
                 State.PLAYING -> playbackTicker.startPlaybackTicker(voiceBroadcastId)
                 State.PAUSED,
                 State.BUFFERING,
                 State.IDLE -> playbackTicker.stopPlaybackTicker(voiceBroadcastId)
             }
-            listeners[voiceBroadcastId]?.forEach { listener -> listener.onStateChanged(playingState) }
+            // Notify state change to all the listeners attached to the current voice broadcast id
+            listeners[voiceBroadcastId]?.forEach { listener -> listener.onPlayingStateChanged(playingState) }
+        }
+    }
+
+    /**
+     * Update the live listening state according to:
+     * - the voice broadcast state (started/paused/resumed/stopped),
+     * - the playing state (IDLE, PLAYING, PAUSED, BUFFERING),
+     * - the potential seek position (backward/forward).
+     */
+    private fun updateLiveListeningMode(seekPosition: Int? = null) {
+        isLiveListening = when {
+            // the current voice broadcast is not live (ended)
+            !currentVoiceBroadcastEvent?.isLive.orFalse() -> false
+            // the player is stopped or paused
+            playingState == State.IDLE || playingState == State.PAUSED -> false
+            // the user has sought
+            seekPosition != null -> {
+                val seekDirection = seekPosition.compareTo(getCurrentPlaybackPosition() ?: 0)
+                when {
+                    // backward
+                    seekDirection < 0 -> false
```

Comment:
Oh, that's what caused one of the issues I mentioned. I guess it's a expected behaviour then.